### PR TITLE
rewrite

### PR DESCRIPTION
**Pull request by bradvogel**
Adding the basic infrastructure for a Mixmax project, including:
* Flow
* Automatic builds via Jenkins
* Jest unit tests
* Sublime project file

There should be no functional changes this commit.

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: 1
```
Comment (bradvogel) [inline config.yml]: remove this

**Comment by jamesmhaley** [comment]
Hey @bradvogel. I see movement!! Let me know if i can help out in the rewrite 😊

**Comment by bradvogel** [comment]
Thanks! Yup, I paused on it, though I'll resume soon.

**Comment by bradvogel** [comment]
Abandoned effort. Closing.

**Pull request closed without merging**